DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata/DirectoryMetadata.py

```diff
@@ -90,7 +96,7 @@ def deleteMetadataField(self, pname, credDict):
         result["Message"] = error + "; " + result["Message"]
     return result
 
-  def getMetadataFields(self, credDict):
+  def getMetadataFields(self, credDict, enableStripping=False):
```

Comment:
In order to keep backward compatibility I would set enableStripping=True
And probably better to use stripVO ?

Reply:
This particular method is used in the handler in which case the VO suffix has to be stripped of and multiple times internally in its class where the suffix has to be present. Flipping the default would require more typing but it would not change anything. The backward compatibility with the single VO system is not maintained by this default but by loading a correct plugin - _DirectoryMetadata_ or _MultiVODirectoryMetadata_ class. In a single VO case the suffix is "stripped" as well, but it is defined to be an empty string so stripping has no effect.

Reply:
I think it's fine to keep it `False` as default there.